docs/developer-handbook.md

```diff
@@ -551,6 +551,15 @@ need to do anything in `@carbon/styles` side.
 
 ## Maintainers
 
+### Public API Snapshot
+
+The entire public api of `@carbon/react` is tracked in one file by iterating
```

Comment:
For consistency, should 'api' here and elsewhere in this pull request be changed to 'API'?